package.json

```diff
@@ -71,7 +71,8 @@
     "examples:install": "node examples/scripts/code-examples.mjs install",
     "examples:build": "node examples/scripts/code-examples.mjs build",
     "examples:test": "node examples/scripts/code-examples.mjs test",
-    "examples:start": "http-server ./examples/tmp"
+    "examples:start": "http-server ./examples/tmp",
+    "build-zip": "npm install && npm run build && cd tmp && npm pack && mv handsontable*.tgz handsontable-$(date +%Y%m%d).tgz && cp ../hf-zip-readme.md README.md && zip handsontable-9.0.0-$(date +%Y%m%d).zip handsontable*.tgz README.md && rm handsontable*.tgz && mv handsontable*.zip .."
```

Comment:
Same as with `hf-zip-readme`, not sure if it's still needed here? It's important to remember to remove it before merging to `develop`.

Reply:
[ref.](https://github.com/handsontable/handsontable/pull/7648#discussion_r616514194)